paddle/fluid/platform/dynload/cublas.h

```diff
@@ -71,8 +73,8 @@ extern void *cublas_dso_handle;
   __macro(cublasDgemm_v2);                \
   __macro(cublasHgemm);                   \
   __macro(cublasSgemmEx);                 \
-  __macro(cublasSgeam_v2);                \
-  __macro(cublasDgeam_v2);                \
+  __macro(cublasSgeam);                   \
+  __macro(cublasDgeam);                   \
```

Comment:
why is this changed?

Reply:
Because there is no API named `cublasDgeam_v2` in cublas. It is a bug before.